src/System.Text.Http/System/Text/Http/HttpRequest.cs

```diff
@@ -22,8 +22,14 @@ public override string ToString()
 
     public struct HttpRequest
     {
+
+#if Windows
         internal static readonly byte[] Cr = new byte[] { (byte)'\r', (byte)'\n' };
         internal static readonly byte[] Cr2 = new byte[] { (byte)'\r', (byte)'\n', (byte)'\r', (byte)'\n' };
+#else
```

Comment:
This is the wrong fix. The fix should be done in the test sources to force a CRLF in the pre-fab HTTP requests since the HTTP spec requires CRLF.